fuzz/meson.build

```diff
@@ -9,6 +11,8 @@ libass_fuzz = executable(
     install: false,
     include_directories: incs,
     dependencies: deps,
+    link_args: get_option('fuzzing-engine'),
+    link_language: 'cpp',
```

Comment:
This now always requires a C++ linker or perhaps depending on what the language declaration earlier does even a full C++ toolchain? Yet, only OSS-Fuzz actually needs C++ linkage making it unnecessarily hard to build the fuzzer for everyone else.

Reply:
> You must use $CXX as a linker, even if your project is written in pure C.

This is requirement from oss-fuzz. Because some "engines" link static libraries that depends on C++ runtime. I don't recall which one exactly needs it, but it's good to be consistent here.

> Yet, only OSS-Fuzz actually needs C++ linkage making it unnecessarily hard to build the fuzzer for everyone else.

Is this a problem? I doesn't think it is common to have C only toolchain setup, especially on fuzzing machines.

I would prefer to keep it simply. Meaning not fragment it to much. I could make it cpp only if oss or add an option. But the question is, does the added complexity is needed to solve any real use-case?

Reply:
C-only toolchains certainly exist and there’s no point in requiring them here for standalone or any other native variants of the utility. Keep in mind that this is also used in testing and to conveniently reproduce crashes.